docs/guides/edgemesh_test_env_guide.md

```diff
@@ -18,19 +18,20 @@ edged:
 > <https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd>
 
 ## Install CNI plugin 
-+ get cni plugin source code of version (0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0) from the github.com. Compile and install.
++ get cni plugin source code of version (0.2.0) from the github.com. Compile and install.
+> It is recommended to use version 0.2.0, which I have tested to ensure stability and availability.
```

Comment:
Better to remove `I`, just `which have been tested...`.
BTW, how about the newest version like 0.5, 0.6...? Noticed that 0.2 is published at 2016

Reply:
ok. And because containerd currently only supports versions prior to 0.4.1. Previously we stated that we can support it to 0.4.1. Recently we found that there is a bug #892.

Reply:
ok, this sound good. Once the bug fixed, let's up to the new version.

Reply:
ok